packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -433,8 +438,16 @@ class _OutlineButtonState extends State<_OutlineButton> with SingleTickerProvide
     ).evaluate(_elevationAnimation);
   }
 
+  // TODO(gspencergoog): implement the real version of this: this is just an
```

Comment:
Does this just mean that this density value has not been defined by the spec yet?